doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -272,17 +300,24 @@ The ECC algorithm support is dependent on one or more Kconfig options enabling c
 ECC driver configurations
 =========================
 
-You can use the following Kconfig options for fine-grained control over which drivers provide ECC support:
+Use the following table to check the ECC algorithm support of each driver:
```

Comment:
Think we should follow the same form for these as well. 'The following table shows ECC algorithim support of each driver'...